site/content/en/docs/admission-check-controllers/provisioning.md

```diff
@@ -56,21 +56,27 @@ Where:
 - **retryStrategy.backoffLimitCount** - indicates how many times ProvisioningRequest should be retried in case of failure. Defaults to 3.
 - **retryStrategy.backoffBaseSeconds** - provides the base for calculating backoff time that ProvisioningRequest waits before being retried. Defaults to 60.
 - **retryStrategy.backoffMaxSeconds** - indicates the maximum backoff time (in seconds) before retrying a ProvisioningRequest. Defaults to 1800.
-- **podSetMergePolicy** - allows to merge similar PodSets into a single PodTemplate used by the ProvisioningRequest. 
+- **podSetMergePolicy** - allows to merge similar PodSets into a single PodTemplate used by the ProvisioningRequest.
+- **podSetUpdates** - allows to update the Workload's PodSets with nodeSelectors based on the successful ProvisioningRequest.
+  This allows to restrict scheduling of the PodSets' pods to the newly provisioned nodes.
+
+#### PodSet merge policy
 
 {{% alert title="Note" color="primary" %}}
 `podSetMergePolicy` feature is available in Kueue v0.12.0 version or newer.
 
-It offers two options: 
-- `IdenticalPodTemplates` - merges only identical PodTemplates 
+It offers two options:
+- `IdenticalPodTemplates` - merges only identical PodTemplates
 - `IdenticalWorkloadSchedulingRequirements`  - merges PodTemplates which have identical fields which are considered for defining the workload scheduling requirements.
```

Comment:
Which fields are exactly considered for defining requirements? Can we have an example of that

Reply:
Done